src/DssSpell.sol

```diff
@@ -52,22 +52,159 @@ contract DssSpellAction is DssAction {
+        DssExecLib.setChangelogAddress("MCD_JOIN_UNIV2AAVEETH_A",  UNIV2AAVEETH_JOIN);
+        DssExecLib.setChangelogAddress("MCD_FLIP_UNIV2AAVEETH_A",  UNIV2AAVEETH_FLIP);
+        DssExecLib.setChangelogAddress("PIP_UNIV2AAVEETH",         UNIV2AAVEETH_PIP);
+
+        // Dai Savings Rate Adjustment
+        PotAbstract(DssExecLib.pot()).drip();
+        DssExecLib.setDSR(ONE_HUNDREDTH_PCT);
+
+        // Remove Permissions for Liquidations Circuit Breaker
+        address flipperMom = DssExecLib.flipperMom();
+        DssExecLib.deauthorize(DssExecLib.flip(PSM_USDC_A_ILK), flipperMom);
+        DssExecLib.deauthorize(DssExecLib.flip(UNIV2DAIUSDC_A_ILK), flipperMom);
+
+        // Fix for Line != sum lines rounding error issue (0.602857457497899800874246318932698818152722680 DAI)
+        VatAbstract vat = VatAbstract(DssExecLib.vat());
+        vat.file("Line", add(vat.Line(), 602857457497899800874246318932698818152722680));
+
+        // Add medianizers to the chain log
+        DssExecLib.setChangelogAddress("MED_ETH", OsmAbstract(DssExecLib.getChangelogAddress("PIP_ETH")).src());
+        DssExecLib.setChangelogAddress("MED_BAT", OsmAbstract(DssExecLib.getChangelogAddress("PIP_BAT")).src());
+        DssExecLib.setChangelogAddress("MED_WBTC", OsmAbstract(DssExecLib.getChangelogAddress("PIP_WBTC")).src());
+        DssExecLib.setChangelogAddress("MED_ZRX", OsmAbstract(DssExecLib.getChangelogAddress("PIP_ZRX")).src());
+        DssExecLib.setChangelogAddress("MED_KNC", OsmAbstract(DssExecLib.getChangelogAddress("PIP_KNC")).src());
+        DssExecLib.setChangelogAddress("MED_MANA", OsmAbstract(DssExecLib.getChangelogAddress("PIP_MANA")).src());
+        DssExecLib.setChangelogAddress("MED_USDT", OsmAbstract(DssExecLib.getChangelogAddress("PIP_USDT")).src());
+        DssExecLib.setChangelogAddress("MED_COMP", OsmAbstract(DssExecLib.getChangelogAddress("PIP_COMP")).src());
+        DssExecLib.setChangelogAddress("MED_LRC", OsmAbstract(DssExecLib.getChangelogAddress("PIP_LRC")).src());
+        DssExecLib.setChangelogAddress("MED_LINK", OsmAbstract(DssExecLib.getChangelogAddress("PIP_LINK")).src());
+        DssExecLib.setChangelogAddress("MED_BAL", OsmAbstract(DssExecLib.getChangelogAddress("PIP_BAL")).src());
+        DssExecLib.setChangelogAddress("MED_YFI", OsmAbstract(DssExecLib.getChangelogAddress("PIP_YFI")).src());
+        DssExecLib.setChangelogAddress("MED_UNI", OsmAbstract(DssExecLib.getChangelogAddress("PIP_UNI")).src());
+        DssExecLib.setChangelogAddress("MED_RENBTC", OsmAbstract(DssExecLib.getChangelogAddress("PIP_RENBTC")).src());
+        DssExecLib.setChangelogAddress("MED_AAVE", OsmAbstract(DssExecLib.getChangelogAddress("PIP_AAVE")).src());
+
+        // bump Changelog version
+        DssExecLib.setChangelogVersion("1.2.7");
     }
 }
 
 contract DssSpell is DssExec {
-    DssSpellAction public spell = new DssSpellAction(true);
+    DssSpellAction public spell = new DssSpellAction();
```

Comment:
For clarity I'd rename this variable to `action`

Reply:
That will conflict with the base namespace.

Reply:
Actually it is a bit weird to have two public variables with the same content in `DssSpell`.
I think this part needs a bit of re-thinking. In the meanwhile we could call it `action_` and not expose it as a `public` variable so we do see it repeated.

Reply:
One easy fix would be to move the description back to the `DssSpell` and just do:
```
constructor() DssExec(
        "2021-02-19 MakerDAO Executive Spell | Hash: 0xedcf17520223556b12c535abe7dfc8c70f4f98d4423119a05a37b92b18048bca",
        block.timestamp + 30 days,
        address(new DssSpellAction())
    ) public {}
```

Reply:
So this was a deliberate design decision. The bytecode at `DssExec` will only include the exec content, so the spell address was exposed here so that we can more easily locate it on-chain if necessary.

I suppose we can analyze the internal transactions to find the location if we need to, but I see this as more of a helper.

Reply:
And tbh I'd rather keep the description out of the constructor and make it an override on the actions. Then the DssExec can just call `spell.description()` if it's called. But that's for next time.

Reply:
Hmmm but the exec code already exposes the `action`, which is the same than what we publicly define as `spell`. So basically `DssSpell` will end up having two public variables `spell` and `action` with the same value.